astropy/time/core.py

```diff
@@ -531,6 +531,125 @@ def value(self):
         """Time value(s) in current format"""
         return self._shaped_like_input(self._time.value)
 
+    def heliocentric_correction(self, skycoord, location=None):
+        """Calculate light travel time correction to the heliocentre
+        
+        This routine the light travel time difference between Earth and
```

Comment:
This sentence just seems to duplicate the title. I'd remove it. (Same for barycentric)